dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/transformed.bnd

```diff
@@ -0,0 +1,28 @@
+#*******************************************************************************
+# Copyright (c) 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/transform.props
+
+Bundle-Name: io.openliberty.microprofile.telemetry.2.0.logging.internal.jakarta
+Bundle-SymbolicName: io.openliberty.microprofile.telemetry.2.0.logging.internal.jakarta
+
```

Comment:
Between this and the `original.bnd` , couldn't the export, private and ds annotation be in the parent `bnd.bnd`?

Reply:
Fixed! Removed the packages as they were redundant.